### PR TITLE
Add CSEM to member and Ajusted order for Intrinsix

### LINKED ISSUE #261: Add CSEM SA as Member
Hello, 
Please add CSEM SA's logo on the OpenHW Group website as Member https://www.openhwgroup.org/
[csem-logo-cmyk.zip](https://github.com/openhwgroup/openhwgroup.org/files/5888163/csem-logo-cmyk.zip)

### PR DESCRIPTION
**Pull request by flora8984461**
Fix #261 

Signed-off-by: Yi Liu <yi.liu@eclipse-foundation.org>

### CODE REVIEW

File: data/members.yml [hunk 1]
```diff
@@ -137,6 +142,11 @@ types:
          title: Imperas
          link: http://www.imperas.com/
          img_width: 100
+       -
+         image: /images/sponsors/intrinsix-logo.png
+         title: Intrinsix Corp
+         link: https://www.intrinsix.com/
+         img_width: 145
        -
```
Comment (flora8984461) [inline data/members.yml:150]: Ah, just found I forgot the alpha order 😅

File: data/members.yml [hunk 2]
```diff
@@ -71,6 +71,11 @@ types:
          title: Codeplay Software
          link: https://www.codeplay.com/
          img_width: 150
+       -
+         image: /images/sponsors/csem-logo-cmyk.png
+         title: CSEM
+         link: https://www.csem.ch/Home
```
Comment (flora8984461) [inline data/members.yml:77]: I wanna confirm the site url as well

Comment (autumnfound) [reply data/members.yml:77]: CESM should be higher in alphabetic order :)

Comment (flora8984461) [reply data/members.yml:77]: @autumnfound Isn't this CSEM? 😅

Comment (flora8984461) [reply data/members.yml:77]: The url is confirmed.

**Review by autumnfound — APPROVED** [review]
D'oh, its CSEM. My eyes no worky. LGTM

**Pull request merged**